gen/definitions/generic/tls_ssl_decryption_policy_definition.yml

```diff
@@ -0,0 +1,192 @@
+name: TLS SSL Decryption Policy Definition
+rest_endpoint: /template/policy/definition/ssldecryption/
+has_version: true
+id_attribute: definitionId
+doc_category: Security Policies
+attributes:
+  - model_name: type
+    type: String
+    value: sslDecryption
+  - model_name: name
+    tf_name: name
+    type: String
+    mandatory: true
+    description: The name of the policy definition.
+    example: Example
+  - model_name: description
+    tf_name: description
+    type: String
+    mandatory: true
+    description: The description of the policy definition.
+    example: My description
+  - model_name: mode
+    tf_name: mode
+    type: String
+    enum_values: [security, unified]
+    description: The policy mode
+    example: security
+  - model_name: type
+    data_path: [definition, defaultAction]
+    tf_name: default_action
+    type: String
+    enum_values: [noIntent, doNotDecrypt, decrypt]
+    description: Default action
+    example: noIntent
+  - model_name: sequences
+    data_path: [definition] 
+    type: List
+    tf_name: network_rules
+    description: List of network rules
+    attributes:
+      - model_name: baseAction
+        type: String
+        tf_name: base_action
+        description: Rule base action
+        enum_values: [noIntent, doNotDecrypt, decrypt]
+        example: doNotDecrypt
+      - model_name: sequenceId
+        type: Int64
+        tf_name: rule_id
+        description: Rule ID
+        example: 4
+      - model_name: sequenceName
+        type: String
+        tf_name: rule_name
+        description: Rule name
+        example: Example
+      - model_name: sequenceType
+        type: String
+        tf_name: rule_type
+        description: Rule type
+        example: sslDecryption
+      - model_name: entries
+        data_path: [match]
+        type: List
+        tf_name: source_and_destination_configuration
+        description: List of network source / destination configuration
+        attributes:
+          - model_name: field
+            type: String
+            tf_name: option
+            description: source / destination option
+            enum_values: [sourceIp, sourcePort, destinationVpn, destinationIp, destinationPort]
+            example: destinationIp
+          - model_name: value
+            type: String
+            tf_name: value
+            description: source / destination option target
+            example: 10.0.0.0/12
+  - model_name: profiles
+    data_path: [definition] 
+    tf_name: url_rules
+    type: List
+    description: List of url rules
+    exclude_test: true
+    attributes:
+      - model_name: name
+        tf_name: rule_name
+        type: String
+        description: Country
+        example: Example
+      - model_name: vpn
+        tf_name: target_vpns
+        type: StringList
+        description: List of VPN IDs
+        example: 1
+      - model_name: ref
+        tf_name: tls_ssl_profile_id
```

Comment:
I think the full name of the resource will be `tls_ssl_profile_policy_definition`, therefore this should be named `tls_ssl_profile_policy_definition_id`. Same for the version attribute below.